go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/grafana/grafana-plugin-sdk-go
 
-go 1.21
+go 1.22.0
```

Comment:
Can you use  just 1.22?

Reply:
So if I use 1.22 it fails(`go: download go1.22 for linux/amd64: toolchain not available`) in the drone build section. https://drone.grafana.net/grafana/grafana-plugin-sdk-go/2582/1/2. Maybe when we migrate to GA then 1.22 might work, I am working on this as well.

Reply:
Probably related to this or similar bug https://github.com/golang/go/issues/68971